internal/controlplane/server.go

```diff
@@ -89,13 +90,14 @@ type Server struct {
 	// We may want to start breaking up the server struct if we use it to
 	// inject more entity-specific interfaces. For example, we may want to
 	// consider having a struct per grpc service
-	ruleTypes     ruletypes.RuleTypeService
-	repos         github.RepositoryService
-	profiles      profiles.ProfileService
-	providers     providers.ProviderService
-	marketplace   marketplaces.Marketplace
-	providerStore providers.ProviderStore
-	ghClient      ghprov.ClientService
+	ruleTypes           ruletypes.RuleTypeService
+	repos               github.RepositoryService
+	profiles            profiles.ProfileService
+	providers           providers.ProviderService
+	marketplace         marketplaces.Marketplace
+	providerStore       providers.ProviderStore
+	ghClient            ghprov.ClientService
+	fallbackTokenClient *gh.Client
```

Comment:
Added this here 1) for testing purposes and 2) because we want to reuse a single `gh.Client` instance to take advantage of the rate limit handling we get from `go-github`.

Reply:
Once I get my ProviderBuilder refactoring merged in, it should be possible to remove this from the Server struct.